yt_dlp/extractor/rudovideo.py

```diff
@@ -0,0 +1,31 @@
+from .common import InfoExtractor
+from ..utils import ExtractorError, traverse_obj
+
+
+class RudoVideoIE(InfoExtractor):
```

Comment:
Since RudoVideo hosts other content such as [VODs](https://rudo.video/vod/bN5AaJ) and [podcasts](https://rudo.video/podcast/cz2wrUy8l0o), this should be preemptively named to reflect that
```suggestion
class RudoVideoLiveIE(InfoExtractor):
```
If you like, you can also implement those, or just leave them for now.